Misc/NEWS.d/next/Library/2022-06-14-22-46-05.gh-issue-83638.73xfGK.rst

```diff
@@ -0,0 +1,6 @@
+Add the :attr:`~sqlite3.Connection.autocommit` attribute
+to :class:`~sqlite3.Connection`
```

Comment:
```suggestion
to :class:`sqlite3.Connection`
```

Otherwise, nothing here actually says what module this is in...oops, probably my bad.